packages/harpguru-core/src/components/rendered-tone/rendered-tone.tsx

```diff
@@ -20,16 +20,21 @@ export const RenderedTone = ({
   isQuestion,
   splitType,
   activeExperienceMode,
+  // TODO: See whether the static sizes might be a solution to this
+  // TODO: I think I might have some trouble here. This component is
```

Comment:
This TODO can be removed. I don't mind the scaling in all instances actually. Everywhere this is used fits with the idea that if you can handle anything scaling smaller then you can handle the majority scaling smaller.